aspnet/Gantt/Context-Menu.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Context Menu | Gantt | ASP.NET Webforms | Syncfusion
-description: context menu
+description: Learn here about context menu in Syncfusion Essential ASP.NET Webforms Gantt Control, its elements, and more.
```

Comment:
Learn here about context menu support in Syncfusion Essential ASP.NET Webforms Gantt Control, its elements, and more.

Reply:
Updated